internal/copywithtimeout/copywithtimeout.go

```diff
@@ -31,11 +31,10 @@ func init() {
 
 // Copy is a wrapper for io.Copy using a timeout duration
 func Copy(dst io.Writer, src io.Reader, size int64, context string, timeout time.Duration) (int64, error) {
-	log := "to EOF"
-	if size > 0 {
-		log = fmt.Sprintf("%d bytes", size)
-	}
-	logrus.Debugf(fmt.Sprintf("hcsshim::copywithtimeout (%s) %s", context, log))
+	logrus.WithField("stdval", context).
+		WithField("size", size).
+		WithField("timeout", timeout.Nanoseconds()).
+		Debug("hcsshim::copywithtimeout - Begin")
```

Comment:
This doesn't seem like the kind of logging we aspire to...

Reply:
In other words, this one seems quite verbose and also probably useless...

Reply:
It doesn't bother me either way. For one off's this seemed to do the trick. As you see in the process/system where we have an object with context we do the keys a different way so we do use the `WithFields`

This could equally be written
```go
fields := logrus.Fields{
    "stdval": context,
    "size": size,
    "timeout": timeout.Nanoseconds(),
}
logrus.WithFields(fields).Debug("hcsshim::copywithtimeout - Begin")
```

Reply:
Moved to the inline style for fields decl.